app/lib/pre_assembly/from_file_manifest/structural_builder.rb

```diff
@@ -5,24 +5,22 @@ module FromFileManifest
     # Updates the Cocina::DROStructural metadata with the new structure derived from a file manifest
     class StructuralBuilder
       # @param [String] reading_order
-      def self.build(cocina_dro:, resources:, content_md_creation_style:, reading_order: 'left-to-right')
-        new(cocina_dro:, resources:,
-            content_md_creation_style:, reading_order:).build
+      def self.build(cocina_dro:, resources:, reading_order: nil)
```

Comment:
Should we be concerned about changing the default `reading_order` here?

Reply:
No, because the reading_order is only used when a simple_book. That same logic (and default) is now applied in https://github.com/sul-dlss/pre-assembly/pull/1226/files#diff-599839b33c4c701e19b9492762fb09fcd056876f9267e04c210ae2aab98ba6f6R160 and passed down as the reading_order.

In other words: previously reading_order was determined based on `reading_order` (left-to-right or right-to-left) and `content_md_creation_style`. Now that same information is carried just by `reading_order` (left-to-right, right-to-left, or nil).